Gemfile

```diff
@@ -58,4 +58,5 @@ gem 'simplecov', :require => false, :group => :test
 group :development, :test do
   gem 'pry'
   gem 'faraday'
+  gem 'byebug'
```

Comment:
i prefer byebug's shortcut methods, namely `s` for step-into, `c` for continue, and `n` for next. I think i once saw a way to to get those to work in pry, but adding byebug was easier.